apps/web/src/components/tx/SignOrExecuteForm/ExecuteForm.tsx

```diff
@@ -187,6 +192,10 @@ export const ExecuteForm = ({
           </Box>
         )}
 
+        {validationError !== undefined && (
+          <ErrorMessage error={new Error(validationError)}>Error validation transaction data</ErrorMessage>
```

Comment:
```suggestion
          <ErrorMessage error={new Error(validationError)}>Error validating transaction data</ErrorMessage>
```

Ideally also the Error object should be memoized.